### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by Barbiero** [petervanderdoes/gitflow-avh#472]
Last commit 3 years ago, over a hundred issues and 30 PRs

Is this project still maintained? Are there any evergreen forks in the wild? funnily enough I got here from reading the nvie/gitflow#6452 issue

I'd appreciate if there was at least a "not actively maintaned" notice on the README.

**Comment by hylke94** [comment]
Indeed, I was wondering why the `-e` flag didn't work on my local machine. But it turns out, it is a [commit from 2019](https://github.com/petervanderdoes/gitflow-avh/commit/16177915cf4c07ce839919eb74d75e66d59973ea), never ever released in [v1.12.4](https://github.com/petervanderdoes/gitflow-avh/commit/a941a2a9245dea5657183b14808402b0be4d3e18).

Any explanation/reason @petervanderdoes ?

**Comment by faandg** [comment]
People are adopting trunk based development over gitflow since it's less bulky/procedural.
I personally don't think nvie's or petervanderdoes's code will still be updated and it's up to them to prove us wrong otherwise.
I'm only here to fetch installation files for our old CI builds.

**Comment by buhtz** [comment]
Can you explain the term "trunk based development" please.

Btw: https://github.com/nvie/gitflow/issues/6484

**Comment by wildernessfamily** [comment]
> People are adopting trunk based development over gitflow since it's less bulky/procedural. I personally don't think nvie's or petervanderdoes's code will still be updated and it's up to them to prove us wrong otherwise. I'm only here to fetch installation files for our old CI builds.

Not true!  I use GitFlow more than any other type of strategy. Yes, on some small, one developer or very small team projects, I use the GitHub strategy. For HIPPA, PCI, any really any other security intensive projects and when I'm the leader, Git Flow type of workflow only! Other times, prior to the project, the team and I sit down, and discuss a strategy. Sometimes we use Github Flow, OR Git Flow, OR GitLab Flow, BUT, more times than not, we create our own workflow/strategy that works best for our project and team. And again, more times than not, it's dynamic. As times goes on, we refine the flow that works best to our needs. But, at the end of the day, many of the features in this package are in use with almost all of our workflows that we use. It's said to see that it has not been updated in years.

**Comment by ChrisJStone** [comment]
I have a fork [here](https://github.com/CJ-Systems/gitflow-cjs) that I will be updating with the open pulls from here and from [nvie](https://github.com/nvie/gitflow) when I have time and also see if I can resolve some of the outstanding issues that are open. Please feel free to resubmit any open PR's you may have or issues.

**Comment by paulwwisl** [comment]
As of 2023

https://github.com/CJ-Systems/gitflow-cjs is active
https://github.com/petervanderdoes/gitflow-avh is still since May 23, 2019
https://github.com/nvie/gitflow is still since Sep 25, 2012
